test/contract-tests/tests/test/amazon/mysql/mysql_test.py

```diff
@@ -0,0 +1,184 @@
+
+    def test_select_succeeds(self) -> None:
+        self.assert_select_succeeds()
+
+    def test_create_item_succeeds(self) -> None:
+        self.assert_create_item_succeeds()
+
+    def test_drop_database_succeeds(self) -> None:
+        self.assert_drop_table_succeeds()
+
+    def test_fault(self) -> None:
+        self.assert_fault()
+
+    @override
+    def _assert_aws_span_attributes(self, resource_scope_spans: List[ResourceScopeSpan], path: str, **kwargs) -> None:
+        target_spans: List[Span] = []
+        for resource_scope_span in resource_scope_spans:
+            # pylint: disable=no-member
+            if resource_scope_span.span.kind == Span.SPAN_KIND_CLIENT and resource_scope_span.span.name == "Execute":
+                target_spans.append(resource_scope_span.span)
+
+        self.assertEqual(
+            len(target_spans), 1, f"target_spans is {str(target_spans)}, although only one value was expected"
+        )
+        self._assert_aws_attributes(target_spans[0].attributes, **kwargs)
+
+    @override
+    def _assert_semantic_conventions_span_attributes(
+        self, resource_scope_spans: List[ResourceScopeSpan], method: str, path: str, status_code: int, **kwargs
+    ) -> None:
+        target_spans: List[Span] = []
+        for resource_scope_span in resource_scope_spans:
+            # pylint: disable=no-member
+            if resource_scope_span.span.kind == Span.SPAN_KIND_CLIENT and resource_scope_span.span.name == "Execute":
+                target_spans.append(resource_scope_span.span)
+        self._assert_semantic_conventions_attributes(target_spans[0].attributes, kwargs.get("sql_command"))
+
+    def _assert_semantic_conventions_attributes(self, attributes_list: List[KeyValue], command: str) -> None:
+        attributes_dict: Dict[str, AnyValue] = self._get_attributes_dict(attributes_list)
+        self.assertTrue(attributes_dict.get("db.statement").string_value.startswith(command))
+        self._assert_str_attribute(attributes_dict, "db.system", self.get_remote_service())
+        self._assert_str_attribute(attributes_dict, "db.name", DATABASE_NAME)
+        self._assert_str_attribute(attributes_dict, "net.peer.name", DATABASE_HOST)
+        self.assertTrue("server.address" not in attributes_dict)
+        self.assertTrue("server.port" not in attributes_dict)
+        self.assertTrue("db.operation" not in attributes_dict)
+
+    @override
+    def _assert_aws_attributes(
+        self, attributes_list: List[KeyValue], expected_span_kind: str = SPAN_KIND_CLIENT, **kwargs
+    ) -> None:
+        attributes_dict: Dict[str, AnyValue] = self._get_attributes_dict(attributes_list)
+        self._assert_str_attribute(attributes_dict, AWS_LOCAL_SERVICE, self.get_application_otel_service_name())
+        self._assert_str_attribute(attributes_dict, AWS_REMOTE_SERVICE, self.get_remote_service())
+        # TODO: https://github.com/aws-observability/aws-otel-dotnet-instrumentation/issues/81
```

Comment:
In [issue #81](https://github.com/aws-observability/aws-otel-dotnet-instrumentation/issues/81), there is `db.statement: SELECT * FROM Blogs`. Why is AWSRemoteOperation set to unknown?

Reply:
The issue happened here: https://github.com/aws-observability/aws-otel-dotnet-instrumentation/blob/main/src/AWS.Distro.OpenTelemetry.AutoInstrumentation/AwsMetricAttributeGenerator.cs#L489
Briefly tested without regex test will show the remote operation